### PR TITLE
Use JavaX Measure Library to handle units

### PR DESCRIPTION
**Pull request by dvdhoz**
Update ConvertUnits class to implement javax measure library to handle unit conversions for metric computes.  Updates made to CumulativeComputable, Total, and MaxAccumDuration. Import UnitsUtil class from Vortex.

### CODE REVIEW

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
+import static systems.uom.common.USCustomary.CUBIC_FOOT;
+import static tech.units.indriya.unit.Units.SECOND;
 
 public final class ConvertUnits {
     private ConvertUnits() {
     }
 
-    public static float convertCfsAcreFeet(int seconds) {
-        return (float) seconds / 43560;
+    public static Unit<?> convertStringUnits(String units) {
```
Comment (joshuarwillis) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java:15]: This method can be declared as package private.

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+        return UnitsUtil.convert(units);
+    }
+
+    public static double getAccumulationConversionFactor(Unit<?> unit) throws IncommensurableException {
```
Comment (joshuarwillis) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java:19]: This method can be declared as package private.

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java
Comment (joshuarwillis) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:1]: Just a general comment to consider. The purpose of the getConversionFactor() method in this class is deducible given the class name, but perhaps renaming it to getAccumulationConversionFactor() may be helpful for additional clarification? I.e. the purpose of this method is to convert from a regular time-series to an accumulation time-series (and not from one unit system to another).

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Total.java [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 
-import static hec.ensemble.stats.ConvertUnits.convertCfsAcreFeet;
+import javax.measure.IncommensurableException;
+import javax.measure.Unit;
 
 public class Total implements Computable, Configurable {
     private static final String DEFAULT_INPUT_UNITS = "cfs";
     Configuration config;
-    private String outputUnit;
+    private Unit<?> unit = null;
```
Comment (joshuarwillis) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Total.java:9]: Is the null initializer of this global variable necessary?

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 
 public  class MaxAccumDuration implements Computable, Configurable {
+    private static final Logger logger = Logger.getLogger(MaxAccumDuration.class.getName());
     private static final String DEFAULT_INPUT_UNITS = "cfs";
-    private static final long DEFAULT_INPUT_DURATION = 3600;  //seconds
-
     Integer accumulatingDuration; //duration in hours
     Configuration config;
-    private String outputUnit;
+    private Unit<?> unit = null;
```
Comment (joshuarwillis) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:13]: Is the null initializer of this global variable necessary?

**Review by joshuarwillis — CHANGES_REQUESTED** [review]
The changes look good @dvdhoz . Just a couple minor suggestions.

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java (hunk 2 above)
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java:19]: Changed to package private

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:1]: Concur.  Changed the method name to getAccumulationConversionFactor

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Total.java (hunk 3 above)
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Total.java:9]: Removed initializer

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java (hunk 4 above)
Comment (dvdhoz) [reply FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java:13]: Removed initializer

**Review by joshuarwillis — APPROVED** [review]

**Pull request merged**